OasisDEX/oasis-borrow
# blur vault cards when there is no input on the vault allowance

![image](https://user-images.githubusercontent.com/3783198/144596814-43cae141-6576-42fc-924f-4d998e5ec58e.png)

https://app.shortcut.com/oazo-apps/story/3149/create-focus-state-when-opening-new-vaults?cf_workflow=500000053&ct_workflow=all
  
## Changes 👷‍♀️
Story
As a user I want to be guided on where I need to focus first when onboarding to a new vault, so that I don't get overwhelmed.

Acceptance Criteria

If the user goes to open a new borrow/multiply/Earn(=guni) vault, the focus is on the sidebar
If the user enters numbers in the sidebar, the rest of the UI lights up.
If the user removes the numbers, the rest of the UI stays lit up.
  
## How to test 🧪
1. go to open a vault - observe the top two and bottom right cards are greyed out
2. enter some value into deposit ETH box - observe the cards are now all showing normally
3. remove the value from the deposit ETH box - observe the cards keep showing normally
    
## Definition of done ✔️
